lib/functional_mixin.js

```diff
@@ -95,6 +108,58 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.swipeLeft = function busterFunctionalMixin_swipeLeft(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
```

Comment:
Yeah, I mean that default number of pixels (actually it sounds close to the idea I described in the other comment). And I'm curious when is it possible for us to perform swiping when screen width is not defined?